%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
% This is just an example/guide for you to refer to when submitting manuscripts to Frontiers, it is not mandatory to use Frontiers .cls files nor frontiers.tex  %
% This will only generate the Manuscript, the final article will be typeset by Frontiers after acceptance.   
%                                              %
%                                                                                                                                                         %
% When submitting your files, remember to upload this *tex file, the pdf generated with it, the *bib file (if bibliography is not within the *tex) and all the figures.
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

%%% Version 3.4 Generated 2022/06/14 %%%
%%% You will need to have the following packages installed: datetime, fmtcount, etoolbox, fcprefix, which are normally inlcuded in WinEdt. %%%
%%% In http://www.ctan.org/ you can find the packages and how to install them, if necessary. %%%
%%%  NB logo1.jpg is required in the path in order to correctly compile front page header %%%

\documentclass[utf8]{FrontiersinHarvard} % for articles in journals using the Harvard Referencing Style (Author-Date), for Frontiers Reference Styles by Journal: https://zendesk.frontiersin.org/hc/en-us/articles/360017860337-Frontiers-Reference-Styles-by-Journal
%\documentclass[utf8]{FrontiersinVancouver} % for articles in journals using the Vancouver Reference Style (Numbered), for Frontiers Reference Styles by Journal: https://zendesk.frontiersin.org/hc/en-us/articles/360017860337-Frontiers-Reference-Styles-by-Journal
%\documentclass[utf8]{frontiersinFPHY_FAMS} % Vancouver Reference Style (Numbered) for articles in the journals "Frontiers in Physics" and "Frontiers in Applied Mathematics and Statistics" 

%\setcitestyle{square} % for articles in the journals "Frontiers in Physics" and "Frontiers in Applied Mathematics and Statistics" 
\usepackage{url,hyperref,lineno,microtype,subcaption}
\usepackage[onehalfspacing]{setspace}
\usepackage[scaled]{helvet}
% \usepackage{helvet}

% \usepackage{palatino}
% \renewcommand\familydefault{\sfdefault} 
\usepackage[T1]{fontenc}
\newcommand{\sprime}{\ensuremath{^\prime}}
\newcommand{\fputr}{5\sprime-UTR}

\linenumbers


% Leave a blank line between paragraphs instead of using \\


\def\keyFont{\fontsize{8}{11}\helveticabold }
\def\firstAuthorLast{Sample {et~al.}} %use et al only if is more than 1 author
\def\Authors{First Author\,$^{1,*}$, Co-Author\,$^{2}$ and Co-Author\,$^{1,2}$}
% Affiliations should be keyed to the author's name with superscript numbers and be listed as follows: Laboratory, Institute, Department, Organization, City, State abbreviation (USA, Canada, Australia), and Country (without detailed address information such as city zip codes or street names).
% If one of the authors has a change of address, list the new address below the correspondence details using a superscript symbol and use the same symbol to indicate the author in the author list.
\def\Address{$^{1}$Laboratory X, Institute X, Department X, Organization X, City X , State XX (only USA, Canada and Australia), Country X \\
$^{2}$Laboratory X, Institute X, Department X, Organization X, City X , State XX (only USA, Canada and Australia), Country X  }
% The Corresponding Author should be marked with an asterisk
% Provide the exact contact address (this time including street name and city zip code) and email of the corresponding author
\def\corrAuthor{Corresponding Author}

\def\corrEmail{email@uni.edu}

\begin{document}
\onecolumn
\firstpage{1}

\title{5\sprime\ untranslated regions of viral mRNAs are repurposing targets} 

\author[\firstAuthorLast ]{\Authors} %This field will be automatically populated
\address{} %This field will be automatically populated
\correspondance{} %This field will be automatically populated

\extraAuth{}% If there are more than 1 corresponding author, comment this line and uncomment the next one.
%\extraAuth{corresponding Author2 \\ Laboratory X2, Institute X2, Department X2, Organization X2, Street X2, City X2 , State XX2 (only USA, Canada and Australia), Zip Code2, X2 Country X2, email2@uni2.edu}


\maketitle


\begin{abstract}

%%% Leave the Abstract empty if your article does not require one, please see the Summary Table for full details.
\section{}
RNA-protein (RNP) complexes regulate nearly every stage of gene expression and play central roles in human disease and viral infection. Despite decades of research, few small molecules have been shown to modulate RNP assemblies with mechanistic understanding. This Perspective argues that the major obstacle is the difficulty of identifying functionally and structurally characterized RNP interfaces within highly dynamic regulatory networks. We propose an integrative discovery framework that combines structure prediction, molecular dynamic simulations, virtual docking, and biophysical validation to enable rational repurposing of FDA approved small molecules.   By focusing on regulatory 5' untranslated regions (5'UTRs) and recurring structural motifs such as bulge loops, we illustrate how minimal RNP constructs and ensemble-based clustering provide a viable path toward targeting functionally characterized RNP complexes. Viral 5'UTRs are presented as representative systems that illustrate how this workflow can reveal molecular vulnerabilities and likely accelerate therapeutic discovery beyond current approaches.


\tiny
 \keyFont{ \section{Keywords:} keyword, keyword, keyword, keyword, keyword, keyword, keyword, keyword} %All article types: you may provide up to 8 keywords; at least 5 are mandatory.
\end{abstract}

\section{Introduction}
Small-molecule (SM) targeting of RNA–protein complexes demands an infusion of innovation to accelerate the discovery of new modalities that both illuminate biological mechanisms and hold therapeutic promise. RNA–protein (RNP) complexes orchestrate essential steps in gene expression under both normal physiological and pathological conditions.  Yet despite decades of research and substantial  investment, the repertoire of SM modulators of RNPs remains limited.  This disparity in part reflects the persistent challenges in identifying RNP interfaces that are functionally relevant, structurally characterized, and amenable to SM intervention. 

Bridging this discovery gap  requires complementary strategies that operate in parallel with sequencing-based approaches that map RNA structure, RNA-binding protein (RBP) interactions, and SM binding sites across the transcriptome.  A majority of expressed human transcripts - estimated at over 50\% - exhibit measurable RBP occupancy \textit{in vivo}, as demonstrated by transcriptome-wide CLIP and RNA interactome capture studies \citep{yeo2020}.  For viruses, the prevalence of RBP-RNA interactions are predicted to be even higher, as both viral and cellular proteins engage viral RNA genomes at nearly every stage of the replication cycle \citep{kim2021,routh2022}. This pervasive and dynamic binding landscape complicates the identification of discrete, druggable RNP interfaces, particularly when interactions are transient, often multivalent, and embedded within larger regulatory networks.

To overcome these challenges, RNP discovery efforts must shift from attempting to screen high-resolution structures of intact assemblies towards focusing on minimal structural fragments that faithfully recapitulate local RNP binding interfaces. Recent advances - exemplified by AlphaFold3 (AF3) - enable modeling of RNP complexes within broader RNA tertiary architectures,  yielding predicted binding interfaces that incorporate stereochemical features imposed by native RNA context and that are often absent from experimental structures of short RNA fragments focused primarily on sequence-specific recognition. When complemented by molecular dynamic  (MD) simulations and biophysical interrogation, these predicted complexes provide a robust foundation for generating dynamic RNP ensembles suitable for virtual  screening.   Together, this hybrid framework links predictive modeling with empirical observation, yielding an experimentally anchored view of RNP structure that captures the transient and allosteric behaviors most relevant to function.



\begin{figure}
    \begin{center}
    \includegraphics[width=\linewidth]{figures/5p-utr-targets.pdf}
    \caption{The \fputr\ constitutes an attack surface replete with targets for SMs perturbing the translation process. Because RBPs are needed to modulate translation for the purposes of the viral life-cycle, any ligand perturbing their affinity for the RNA-element they interact with has potential for antiviral activity. An SM may do this by binding the RBP in an allosteric site (Target class 1), competing directly for the RNA-element with the RBP (Target class 2), perturbing modifying enzymes that act on the \fputr\ (Target class 3) or RBP (target class 4) or finally SMs that bind across the RNA-protein interface, preorganizing it to prefer binding to one factor over another (Target class 5).}\label{fig:5p-utr-targets}
    \end{center}
\end{figure}


Regulatory untranslated regions (UTRs), particularly 5\sprime\ UTRs, occupy a unique position at the intersection of RNA structure, protein binding, and functional output (Fig. \ref{fig:5p-utr-targets}). 
\fputr s integrate conserved secondary and tertiary structural motifs, chemical modifications, and dynamic protein interactions to control ribosome recruitment, translation efficiency, and cellular responses to stress and signaling cues. Dysregulation of \fputr-mediated control is increasingly implicated in cancer, neurodegeneration, and viral pathogenesis, underscoring their biological and therapeutic relevance \citep{hentze2012}.

Among both cellular and viral RNAs, 5'UTRs are enriched in structural elements such as stem loops,   junctions, internal loops, and bulges that serve as docking platforms for RBPs and other regulatory factors. These features position 5'UTRs as natural focal points for identifying discrete, mechanistically interpretable RNP interfaces within otherwise complex transcript-wide interaction networks.

Within these regulatory regions, bulge loops emerge as recurrent and functionally significant RNA structural motifs.  Local deviations from canonical helices expose unpaired nucleotides that frequently serve as recognition elements for RBPs and other ligands  \citep{patel2000}.  The conformational plasticity of bulge loops enables diverse binding modes, allowing them to accommodate proteins, SMs, and other RNA elements with distinct stereochemical features.

Surveys of experimentally determined RNA structures indicate that bulges and internal loops together constitute a substantial fraction (about 20-30\%) of local non-canonical structural motifs. 
Large-scale annotation efforts based on curated RNA structures, including bpRNA, consistently identify bulges and internal loops as among the most prevalent non-WC elements across structured RNAs \citep{hendrix2018}.
In viral genomes, where compact coding constraints favor conserved multifunctional RNA elements, bulges are particularly enriched with UTRs and internal ribosome entry sites (IRES), where they functionas adaptable platforms for hist and viral factor recruitment (Nat Rev Mol Cell Biol 19, 158–174 (2018). https://doi.org/10.1038/nrm.2017.103).

Consistent with this structural prevalence, curated RNA-ligand datasets reveal that bioactive SMs disproportionately engage RNAs containing bulges, internal loops, and junctions rather than fully base-paired helices.  \citep{hargrove2022}. These motifs present unique stereochemical environments characterized by exposed bases, backbone flexibility, and transient pocket formation, making them attractive targets for SM recognition and RNP modulation.

Here, we advance a conceptual framework integrating structure prediction, molecular simulation, and experimental validation to evaluate the druggability of RNP complexes. 
We discuss this framework as applied to the \fputr\ of Enterovirus A-71 (EV-A71) as a representative example.  
This region contains an internal ribosome entry site (IRES) that engages more than 20 host RBPs and a viral-derived small RNA to modulate translation.  
Perturbations that weaken these RBP-IRES interactions significantly reduce viral titers by compromising cap-independent translation.  
Multiple RNA-focused screens have identified small molecules that suppress EV-A71 replication, with DMA-135 emerging as a selective modulator of a key IRES-protein interface \citep{tolbert2020,Davila-Calderon2024}. 
These observations underscore the potential for integrative structural and computational methodologies to pinpoint mechanistic vulnerabilities in viral RNA regulation and accelerate the rational repurposing of clinically relevant compounds for antiviral intervention.

\section{Viral \fputr s are an attack surface for antiviral therapies}

% Conceptually, there are at least five types of interactions present in functional \fputr s that could be modulated by noncovalent binders---of these, targeting binders to the RNP interface or interaction sites on the RNA itself are most appealing (Sudeshi).

% RNA and RNA-protein interaction sites also exist in viral RNAs, such as the \fputr of EV71.
The \fputr s of viral RNAs play a crucial role at different stages of a virus's lifecycle such as regulating translation, replication, and packaging.
This is especially true if the RNA in question is the viral genome, as is the case for positive single-stranded RNA viruses.
The secondary structures of many viral \fputr s are well conserved, folding into multiple stem-loops (SLs) and consist of critical regulatory elements like the internal ribosome entry site (IRES) for cap-independent translation, signals for genome replication, and specific sites recognized by viral proteins for packaging into new virions.
Different structured as well as unstructured regions within viral \fputr s harbor specific binding sites for cellular RNA-binding proteins (RBPs) and miRNAs that modulate viral replication.
Since viral \fputr\ plays a pivotal role by recruiting and repurposing host RBPs to complement for their small genome sizes (2-30 kb) and limited coding capabilities, it becomes an attractive target for antiviral drugs that disrupt these vital host RBP-viral RNA interactions \citep{rouskin2024}.

\subsection{EV-A71}\label{s:5p-targets}

A well-known example is the EV-A71 \fputr, a control hub for EV-A71 genome replication and translation.
It consists of the cloverleaf structure (SLI) and the IRES (SLII-VI) where cellular proteins also referred to as IRES trans-acting factors (ITAFs) bind to specific region(s) within the IRES to either positively or negatively regulate IRES-mediated translation while another set of cellular proteins bind to SLI to regulate viral genome replication.
So far, more than 20 cellular RBPs have been identified to interact with EV-A71 \fputr\ to regulate viral replication \citep{tolbert2024a}. 
To date, SLII is the only well-characterized structure within the EV-A71 \fputr\ where specific binding sites of several of its ITAFs have been identified.
RBPs hnRNP A1 and AUF1 are known to bind at the internal bulge of SLII to enhance and suppress IRES-mediated translation, respectively.
This bulge has been successfully targeted by a small molecule antiviral DMA-135 that binds RNA and strengthens the host AUF1-viral SLII RNA interaction, thus shifting the SLII regulatory axis towards translation repression \citep{tolbert2020}.
This demonstrates the potentially therapeutic effect perturbation of specific host RBP-viral \fputr\ interactions with SMs can have.

\subsection{HIV}

The HIV-1 \fputr\ is also critical for multiple steps in the viral life cycle, where it interacts with host factors to regulate multiple processes including viral genome transcription and splicing, reverse transcription, and genome packaging to form new virions.
Hence, key regulatory elements within the \fputr\ such as the TAR stem-loop, primer binding site (PBS), dimerization initiation site (DIS), and packaging signal (\(\Psi\)) are attractive drug target sites for HIV-1 inhibition \citep{hu2024,roebuck1998,brakier-gingras2012}. 
To date, antisense oligonucleotides (ASOs), small interfering RNAs (siRNAs) and RNA aptamers have been exploited as antiviral strategies to target HIV-1 \fputr, while there are no FDA-approved small molecule drugs \citep{kjems2007,berzal-herranz2014,mergia2009}. So far, several small molecule RNA ligands targeting the TAR hairpin have been identified as inhibitors of viral replication \citep{duca2019}. 

\subsection{SARS-CoV-2}

Because it is a positive strand RNA virus, the \fputr\ of SARS-CoV-2 also plays critical roles in viral genome replication and translation, though these are as yet more poorly understood.
To date, there are no FDA-approved drugs targeting the \fputr\ of SARS-CoV-2.
However, some ASOs and SMs that bind to specific secondary structures within it demonstrate antiviral properties. 
For example, ASOs that bind to the SL1 and SM amiloride derivatives that bind to SL4, SL5a, and SL6 suppress SARS-CoV-2 replication \citep{wang2021,hargrove2021}. 
There is also some evidence describing host RBP-viral RNA interactions based on conserved and \fputr\ structural elements. For example, \citet{pei2023} demonstrated that RBM24 binds within SL4 stem-loop in the SARS-CoV-2 \fputr\ to suppress SARS-CoV-2 replication.
Using a neural net trained to predict RBP binding sites from secondary structures, \citet{zhang2021} predicted several host RBPs such as IGF2BP1, hnRNP A1, hnRNPK, and TIA1, to bind different SLs in the SARS-CoV-2 \fputr.
Hence, there is promise in identifying modulators of these host RBP-viral RNA interactions toward successful antiviral therapies.

\section{There is evidence that FDA approved molecules bind to RNAs and modulate RNA-protein complexes}

% The activity of RNA is tightly regulated by RNA binding proteins (RBPs) through sequence or structural recognition \citep{Gerstberger2014,Dominguez2018}. Dysregulation or off-target binding of RBPs is associated with neurodegenerative diseases \citep{Bryce-Smith2025,Zeng2025}, cancer \citep{Bell2013}, and developmental disorders \citep{Darnell2011}. 
% Amyotrophic lateral sclerosis (ALS) is a neurodegenerative disease characterized by an abnormal accumulation of fused in sarcoma (FUS) protein in the cytoplasm. 
% To minimize the accumulation of cytoplasmic FUS, existing cellular FUS proteins enagage pre-mRNA, producing a feedback loop in the presence of excess FUS protein, thus halting the expression of new FUS proteins \citep{Ayala2011}. 
% Consequently, the disruption of the FUS-pre-messenger RNA (mRNA) interaction exacerbates ALS progression. Another neurodegenerative disease, frontotemporal dementia (FTD) is characterized by the depletion of the mRNA splicing protein, TDP-43, in the nucleus of neurons. 
% Recent studies have described an alternate functional role for TDP-43; TDP-43 interacts with GU rich repeat sequences in mRNAs to modulate cleavage and alternative polyadenylation (APA) \citep{Bryce-Smith2025,Zeng2025}. 
% Improper regulation of APA initiates a cascade of events that results in the loss of functional proteins for axonal regeneration/repair \citep{Melamed2019}. These examples highlight the integral role of proper ribonucleoprotein (RNP) regulation in the maintenance of optimal cellular function.

% Enhanced specificity of RBP interactions can be achieved by cooperative binding of RNA recognition motifs (RRM) within RBPs to their cognate RNA binding sites \citep{Mackereth2011,Hennig2015,Duszczyk2022}. 
Endogenous small molecules such as adenosine triphosphate (ATP), guanosine triphosphate (GTP), and S-adenosylmethionine (SAM) regulate various cellular processes such as RNA localization, translation, and mRNA processing via mediation of RBP-RNA interactions \citep{Miao2025}. 
Non-endogenous small molecules have also emerged as an attractive tool for modulating RBP-RNA interactions. 
In 2020, the splice modulator risdiplam was approved by the Food and Drug Administration (FDA), serving as the only non-ribosomal RNA-targeting FDA approved small molecule to date \citep{Mullard2020,Ratni2018,Ratni2018}. 
Risdiplam stabilizes the interaction between U1 small nuclear ribonucleoprotein and survival motor neuron (SMN) mRNA, leading to replenished SMN2 protein levels for proper maintenance of motor neurons \citep{Ratni2018,Ratni2021}. 
Similarly, the amiloride DMA-135 stabilizes the RNP interaction between AUF1 and stem loop 2 of EV-A71 internal ribosomal entry site (IRES), thereby repressing cap-independent translation of important viral proteins \citep{Davila-Calderon2020,Davila-Calderon2024}. 
However, the expediency in this approach is reduced by the number of RNA-targeting small molecules that have been approved by the FDA.
 
The dearth in RNA-targeting FDA-approved small molecules can partly be ascribed to the conformational heterogeneity of RNAs, as well as the relatively fewer RNA structures deposited in the Protein Data Bank (approximately 26 protein structures for every single RNA structure).
Encouragingly, recent RNA-small molecule screening campaigns have identified some protein inhibitors that bind RNAs as well. 
A few examples include: mitoxantrone, a topoisomerase II inhibitor that binds to the oncogenic pre-miR-21 RNA \citep{Velagapudi2018}. 
Another example is the selective estrogen receptor modulator (SERM) raloxifene that also binds to the \fputr\ of the hHBV \textbf{\(\epsilon\)} RNA \citep{LeBlanc2022}. 
In fact, it was previously shown that several protein-binding small molecules possess significant chemical structure similarities to RNA-binders, hinting at mechanisms by which these drugs confer side-effects \citep {Fang2023}. 

Levofloxacin is an FDA-approved oral antibiotic that functions as an inhibitor of the bacterial enzymes, DNA gyrase and topoisomerase IV \citep{Croom2003}. 
However, a few common side effects associated with the consumption of this drug include: nausea, neuropathy, and difficulty breathing. 
In one case study, \citet{Fang2023} demonstrated that Levofloxacin has pervasive interactions throughout the transcriptome. 
In addition, they demonstrated that RNA-Levofloxacin interactions were not only biochemically driven, but had functional relevance in the context of cellular output. 
Levofloxacin interacts with a G4 motif to compromise translation of the Y-box binding protein 1 (YBX1) mRNA, possibly contributing to the side effects observed with Levofloxacin consumption. 
Off-target binding of Levofloxacin to the \fputr\ of YBX1 mRNA alters the structural dynamics of certain regions in the RNA, potentially modulating the loading of translation initiation factors \citep{Fang2023}. 
Similarly, FDA-approved aminoglycosides such as paromomycin function by binding to ribosomal RNAs to repress or compromise the expression of essential bacterial proteins \citep{Fourmy1996,Recht1996,Vicens2001}. 
The direct conclusion implies potential repurposing of Levofloxacin to target pathogens with G4 motifs that are pivotal to their life cycles. 
Taken together, however, these instances of SM and FDA approved molecules interacting with RNP targets implies repurposing versus \fputr s as a more general strategy to combat viral pathogens.

\section{There is a path to virtually screening many potential targets within a \fputr\ of interest}

\begin{figure}
    \begin{center}
    \includegraphics[width=0.8\linewidth]{figures/workflow-virtual-expt-5putr-screen.pdf}
    \caption{An illustration of the workflow we propose. Using the literature, RNA-elements and their protein binding partners in the \fputr s of disease relevant mRNAs are identified. The RNA elements are grafted into a designed fragment using computational tools. These models are validated using rapid experimental means such as NMR, and simulated with molecular dynamics to model the ensemble of the target. Using pocket finding tools and ensemble virtual screening with a library of approved molecules, we prioritize SMs to validate experimentally. SMs that show biophysical and infectivity assay activity become antiviral treatment options because of their approved nature.}\label{fig:workflow}
    \end{center}
\end{figure}

% The process of discovering novel therapies has historically been a difficult task, thus highlighting the need for creating pipelines that not only fast-track the discovery process, but also increase the probability of successful outcomes. 
% A central objective of this study is to develop a broadly applicable framework for uncovering approved therapeutics with previously unrecognized biological or functional relevance in disease settings distinct from their initial clinical indications. 
% Employing a virtual fragment-based screening workflow has the potential to significantly accelerate the detection of such repurposing candidates. 
% Previous studies have shown that many small molecules primarily bind/target single-stranded regions (i.e. bulge) of RNA elements with high degrees of structure [Ref], and because of this UTRs are ideal candidates for targeting due to their high degree of structure, and large number of proteins interacting at these structured RNA elements which contribute to the regulation (translation and turnover) of the RNA transcript [Ref]. 
% Modeling of RNP complexes can be accomplished using tools such as Alphafold3, and for improved accuracy of RNP models, a fragment based approached can be implemented where minimal constructs of RNA elements are used during the modeling process to increase the likelihood of capturing native RNA-protein interfaces. 
% Further optimization efforts can be made by running molecular dynamics simulations on AF3 outputs for conformational ensemble generation followed by docking, and experimental validation.

Taking EV-A71 as a model system, the \fputr\ is predicted to fold into six individual stem-loops, all of which have multiple interacting protein partners that contribute to the overall regulation of the viral lifecycle.
As discussed in Section \ref{s:5p-targets}, the first stem-loop (SLI) has been shown to be primarily responsible/implicated in viral replication \citep{shih2009,shih2014,tolbert2024a}. 
The remaining stem-loops (SLII-SLVI) collectively make up the internal ribosomal entry site and control viral protein production by modulating its interactions with various protein partners \citep{sarnow2003,tolbert2017,Davila-Calderon2020,tolbert2024a}. 
In its entirety, the \fputr\ binds more than 20 proteins that fall into various protein classes such as heterogeneous nuclear ribonucleoprotein (hnRNPs), enzymes, and transcription factors \citep{shih2009,Davila-Calderon2020}.
Thus, by targeting specific RNP complexes, downstream aspects of the viral lifecycle that are related to a given RNP complex can be altered. 

A significant barrier to discovering novel SMs capable of inhibiting EV-A71 infection is the limited availability of experimentally determined, high-resolution structures of intact RNP complexes that regulate viral replication. 
We hypothesize that AF3 can generate accurate sub-models of these RNP assemblies in which the predicted structures recapitulate the salient features of the  binding interface. 
These AF3-generated models can then be refined using enhanced sampling MD simulations to produce ensembles that explore alternative local binding geometries suitable for virtual screening.
In this framework, AF3 prediction functions primarily as a source of structurally informed  starting coordinates that appropriately constrain the binding interface.

This hypothesis is motivated by the well-established observation that many RBPs recognize short, often degenerate sequence motifs situated within unpaired or conformationally flexible regions of RNA secondary structure (REF). 
Currently, the Protein Data Bank PDB contains on the order of 5,000 distinct protein-RNA complexes, approximately 2000 of which represent non-ribosomal RNPs featuring RBPs bound to short cognate RNA segments. 
These structures offer a rich training set from which AF3 can learn fundamental determinants of interfacial protein-RNA recognition.

Building on this rationale, we propose to simplify the modeling task by restricting AF3 predictions to the minimal  
RNP structural fragments necessary to recapitulate the local binding interface.  
This strategy rests on the assumption that for SM virtual screening, the the local stereochemical environment primarily dictates docking accuracy and efficacy.  
Although distal structural interactions contribute to the overall binding affinity and global features of molecular recognition, they are generally less critical for identifying SM modulators that act at a specific RNP interface.

Figure \ref{fig:workflow} illustrates this strategy using the EV-A71 SLII domain and one of its cognate RBPs as a representative example. 
The bulge loop of SLII serves as the primary recognition element for both hnRNP A1 and AUF1, which bind competitively to this site to modulate IRES-dependent translation efficiency (REF). 
Several crystal and NMR structures exist (PDB codes =) for each protein bound to short RNA fragments resembling the SLII bulge motif (5\sprime -AAUAGCA-3\sprime), underscoring the suitability of this region for structure-guided modeling.  
An additional advantage of this sub-fragment approach is that isotopically labeled protein and RNA constructs can be made in parallel, enabling straightforward NMR chemical shift mapping to validate the local RNP binding interface.

Our approach to generating an AF3 model of the hnRNP A1-SLII complex suitable for virtual screening involves designing a truncated SLII construct that retains the bulge loop and three base pairs flanking it on each side (SLII-sf). 
Because the apical loop is not the target of interest, we propose replacing it with a GNRA or UNCG tetraloop, both of which are extensively represented in the PDB and provide stable, well-defined structural caps that are not known binding sites for RBPs. For hnRNP A1, the tandem RRM domains constitute the RNA-binding specificity module; therefore, the UP1 fragment will be paired with the SLII-sf as the input for AF3-guided complex modeling.   
This targeted approach increases the likelihood that the predicted complex captures the native binding mode, as most RNA binding proteins recognize sequence-specific motifs presented within single-stranded regions of structured RNAs \citep{patel2000,hentze2012,yeo2020}.  
By isolating the relevant RNA element as a modular bulge fragment, we focus on the interaction surface where the RBP engages the target through its canonical B-sheet interface.  
Matched isotopically-labeled protein and RNA constructs should be prepared in parallel, facilitating NMR based validation of the predicted binding interface.

To perform a virtual screen, we'd first run Molecular Dynamics simulations (MD) of these complexes looking for pockets.
Although MD is a medium-low throughput technique, it can help validate AF2 models since it is more physically grounded; consistency in key contacts across the model and simulated ensemble lend credibility to both \citep{hennig2025,vlamos2025,gsponer2025}.
Using either adaptive sampling, or some other exploratory enhanced sampling such as Gaussian-accelerated MD, provides a rough sketch of the ensemble of the complex on a 'several weeks' timeline for common compute resources \citep{defabritiis2016,miao2021}.
Shorter timescale simulations also permit the detection of cryptic pockets, which can often appear within hundreds of nanoseconds of replicate simulation \citep{bowman2023}.
For identifying pockets, ligsite or f-pocket can be used, as well as the RNA specific update proposed by \citet{weeks2025} \citep{barnickel1997,tuffery2009}.
Initial simulations in the group have found conformations that are interesting, and a starting model that is plausible, when this workflow is applied to EV71 SLII in complex with UP1 starting from an AF3 model (see Fig. \ref{fig:pocket-teaser}).

\begin{figure}
    \centering
    \includegraphics[width=\linewidth]{figures/pocket-teaser.eps}
    \caption{Fragment design and simulated conformations for the SLII bulge from EV71A, in complex with the UP1 domain of hnRNPA1. Panel A shows the AF3 model we used to initialize our simulations. Here, the full-length RNA is in magenta, RRM1 of the simulated conformation is in yellow, the interdomain linker is in grey, and RRM2 is in blue. Panel B shows a relaxed structure from these simulations with PDB 4YOE as a reference.  4YOE is in cyan, with its RNA backbone shown in orange and bases in cyan, with the simulated conformation colored as in Panel A. Panels C and D show potentially useful pockets revealed in simulation; here they are highlighted by black ovals. C shows a pocket on the 90-degree rotation downward of the conformation from B. Panel D is a novel pocket that emerges between the interdomain linker and the body of RRM2 in some of the states observed in simulation.}
    \label{fig:pocket-teaser}
\end{figure}
Docking would then be performed versus the ensembles that exhibit pockets using the Broad drug repurposing hub database, and using PopShift to aggregate the docking results \citep{golub2017,bowman2024}.
Virtual screening versus an ensemble is important because complexes involving RNA result in dynamic pockets \citep{al-hashimi2019}.
Even typically `rigid' proteins exhibit pocket dynamics \citep{gilson2017, smith2018}.
PopShift addresses some issues with ensemble docking by using a discretized model of the ensemble (usually a Markov State Model (MSM)) to organize and reduce the number of receptor conformations to be docked to \cite{bowman2024}.
Because PopShift scores which receptor conformations are preferred by different ligands, it can be used to understand rudimentary allosteric preferences sampled in the ligand-free ensemble, per the conformational selection hypothesis \citep{edelstein2011}.

To experimentally validate the simulation results, we will employ a nuclear magnetic resonance (NMR)–based approach, a well-established method for characterizing interactions between RNA, proteins, and small molecules \citep{wuthrich1986nmr};\citep{RN90}. As shown in Figure 2, selective isotopic labeling of both components of the protein–RNA complex enables the simultaneous acquisition of interaction data for each binding partner from a single sample \citep{RN92}. Moreover, analysis of concentration-dependent chemical shift perturbations affords site-specific insight into intermolecular contacts, facilitating precise localization of the amino acid residues and nucleotides that collectively constitute the binding interface \citep{RN91}. Although the experimental workflow depicted in Figure 2 outlines an NMR-based approach, this component of the pipeline is readily adaptable and may be replaced with any suitable biophysical method capable of interrogating the macromolecular system of interest.

\section{Discussion and Conclusion}

Here, we propose that novel RNP targets for therapeutic intervention can be identified far more efficiently by integrating AlphaFold3 predictions with rigorous biophysical characterization. The core innovation of the workflow is its focus on predicting minimal RNP structural fragments that accurately recapitulate the native binding interface, thereby constraining the search space to the most relevant stereochemical features. By coupling these fragments to adaptive-sampling molecular dynamics, we sample a broader fraction of the conformational landscape and obtain well-resolved ensemble states. Virtual docking can then be performed on representative cluster centers, enabling an ensemble-based view of ligandability rather than relying on a single static structure. In parallel, NMR chemical shift perturbations provide a rapid and sensitive experimental readout to validate interface fidelity and confirm ligand-induced perturbations in the local RNA environment.

This integrated strategy has the potential to substantially shorten the time required to identify small molecules capable of modulating RNP interactions. Because functional specificity is likely to arise not from a single lock-and-key interaction between small molecules and RNA, but from shifting the equilibria of broader RNP  networks, we anticipate that this minimal fragment, ensemble-guided approach will generalize well beyond the systems discussed here. Looking forward, such workflows could form the basis of a new discovery paradigm - one in which predictive modeling, conformational sampling, and targeted biophysics operate synergistically to illuminate previously inaccessible RNP interfaces. As structural prediction engines and biophysical methods continue to mature, this strategy is poised to accelerate the development of therapeutics that modulate RNP assemblies across diverse disease contexts.


\section*{Conflict of Interest Statement}
%All financial, commercial or other relationships that might be perceived by the academic community as representing a potential conflict of interest must be disclosed. If no such relationship exists, authors will be asked to confirm the following statement: 

The authors declare that the research was conducted in the absence of any commercial or financial relationships that could be construed as a potential conflict of interest.

\section*{Author Contributions}

The Author Contributions section is mandatory for all articles, including articles by sole authors. If an appropriate statement is not provided on submission, a standard one will be inserted during the production process. The Author Contributions statement must describe the contributions of individual authors referred to by their initials and, in doing so, all authors agree to be accountable for the content of the work. Please see  \href{https://www.frontiersin.org/about/policies-and-publication-ethics#AuthorshipAuthorResponsibilities}{here} for full authorship criteria.

\section*{Funding}
Details of all funding sources should be provided, including grant numbers if applicable. Please ensure to add all necessary funding information, as after publication this is no longer possible.

\section*{Acknowledgments}
This is a short text to acknowledge the contributions of specific colleagues, institutions, or agencies that aided the efforts of the authors.

\section*{Supplemental Data}
 \href{http://home.frontiersin.org/about/author-guidelines#SupplementaryMaterial}{Supplementary Material} should be uploaded separately on submission, if there are Supplementary Figures, please include the caption in the same file as the figure. LaTeX Supplementary Material templates can be found in the Frontiers LaTeX folder.

\section*{Data Availability Statement}
The datasets [GENERATED/ANALYZED] for this study can be found in the [NAME OF REPOSITORY] [LINK].
% Please see the availability of data guidelines for more information, at https://www.frontiersin.org/about/author-guidelines#AvailabilityofData

\bibliographystyle{Frontiers-Harvard} %  Many Frontiers journals use the Harvard referencing system (Author-date), to find the style and resources for the journal you are submitting to: https://zendesk.frontiersin.org/hc/en-us/articles/360017860337-Frontiers-Reference-Styles-by-Journal. For Humanities and Social Sciences articles please include page numbers in the in-text citations 
%\bibliographystyle{Frontiers-Vancouver} % Many Frontiers journals use the numbered referencing system, to find the style and resources for the journal you are submitting to: https://zendesk.frontiersin.org/hc/en-us/articles/360017860337-Frontiers-Reference-Styles-by-Journal
\bibliography{repurposing-vs-5p-utr,repurposing-vs-5p-utr-lgs}

%%% Make sure to upload the bib file along with the tex file and PDF
%%% Please see the test.bib file for some examples of references

% \section*{Figure captions}

%%% Please be aware that for original research articles we only permit a combined number of 15 figures and tables, one figure with multiple subfigures will count as only one figure.
%%% Use this if adding the figures directly in the mansucript, if so, please remember to also upload the files when submitting your article
%%% There is no need for adding the file termination, as long as you indicate where the file is saved. In the examples below the files (logo1.eps and logos.eps) are in the Frontiers LaTeX folder
%%% If using *.tif files convert them to .jpg or .png
%%%  NB logo1.eps is required in the path in order to correctly compile front page header %%%

% \begin{figure}[h!]
% \begin{center}
% \includegraphics[width=10cm]{logo1}% This is a *.eps file
% \end{center}
% \caption{ Enter the caption for your figure here.  Repeat as  necessary for each of your figures}\label{fig:1}
% \end{figure}

% \setcounter{figure}{2}
% \setcounter{subfigure}{0}
% \begin{subfigure}
% \setcounter{figure}{2}
% \setcounter{subfigure}{0}
%     \centering
%     \begin{minipage}[b]{0.5\textwidth}
%         \includegraphics[width=\linewidth]{logo1.eps}
%         \caption{This is Subfigure 1.}
%         \label{fig:Subfigure 1}
%     \end{minipage}  
   
% \setcounter{figure}{2}
% \setcounter{subfigure}{1}
%     \begin{minipage}[b]{0.5\textwidth}
%         \includegraphics[width=\linewidth]{logo2.eps}
%         \caption{This is Subfigure 2.}
%         \label{fig:Subfigure 2}
%     \end{minipage}

% \setcounter{figure}{2}
% \setcounter{subfigure}{-1}
%     \caption{Enter the caption for your subfigure here. \textbf{(A)} This is the caption for Subfigure 1. \textbf{(B)} This is the caption for Subfigure 2.}
%     \label{fig: subfigures}
% \end{subfigure}

%%% If you don't add the figures in the LaTeX files, please upload them when submitting the article.
%%% Frontiers will add the figures at the end of the provisional pdf automatically
%%% The use of LaTeX coding to draw Diagrams/Figures/Structures should be avoided. They should be external callouts including graphics.

\end{document}
