%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
% This is just an example/guide for you to refer to when submitting manuscripts to Frontiers, it is not mandatory to use Frontiers .cls files nor frontiers.tex  %
% This will only generate the Manuscript, the final article will be typeset by Frontiers after acceptance.   
%                                              %
%                                                                                                                                                         %
% When submitting your files, remember to upload this *tex file, the pdf generated with it, the *bib file (if bibliography is not within the *tex) and all the figures.
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

%%% Version 3.4 Generated 2022/06/14 %%%
%%% You will need to have the following packages installed: datetime, fmtcount, etoolbox, fcprefix, which are normally inlcuded in WinEdt. %%%
%%% In http://www.ctan.org/ you can find the packages and how to install them, if necessary. %%%
%%%  NB logo1.jpg is required in the path in order to correctly compile front page header %%%

\documentclass[utf8]{FrontiersinHarvard} % for articles in journals using the Harvard Referencing Style (Author-Date), for Frontiers Reference Styles by Journal: https://zendesk.frontiersin.org/hc/en-us/articles/360017860337-Frontiers-Reference-Styles-by-Journal
%\documentclass[utf8]{FrontiersinVancouver} % for articles in journals using the Vancouver Reference Style (Numbered), for Frontiers Reference Styles by Journal: https://zendesk.frontiersin.org/hc/en-us/articles/360017860337-Frontiers-Reference-Styles-by-Journal
%\documentclass[utf8]{frontiersinFPHY_FAMS} % Vancouver Reference Style (Numbered) for articles in the journals "Frontiers in Physics" and "Frontiers in Applied Mathematics and Statistics" 

%\setcitestyle{square} % for articles in the journals "Frontiers in Physics" and "Frontiers in Applied Mathematics and Statistics" 
\usepackage{url,hyperref,lineno,microtype,subcaption}
\usepackage[onehalfspacing]{setspace}
\usepackage[scaled]{helvet}
% \usepackage{helvet}

% \usepackage{palatino}
% \renewcommand\familydefault{\sfdefault} 
\usepackage[T1]{fontenc}
\newcommand{\sprime}{\ensuremath{^\prime}}
\newcommand{\fputr}{5\sprime-UTR}

\linenumbers


% Leave a blank line between paragraphs instead of using \\


\def\keyFont{\fontsize{8}{11}\helveticabold }
\def\firstAuthorLast{Sample {et~al.}} %use et al only if is more than 1 author
\def\Authors{First Author\,$^{1,*}$, Co-Author\,$^{2}$ and Co-Author\,$^{1,2}$}
% Affiliations should be keyed to the author's name with superscript numbers and be listed as follows: Laboratory, Institute, Department, Organization, City, State abbreviation (USA, Canada, Australia), and Country (without detailed address information such as city zip codes or street names).
% If one of the authors has a change of address, list the new address below the correspondence details using a superscript symbol and use the same symbol to indicate the author in the author list.
\def\Address{$^{1}$Laboratory X, Institute X, Department X, Organization X, City X , State XX (only USA, Canada and Australia), Country X \\
$^{2}$Laboratory X, Institute X, Department X, Organization X, City X , State XX (only USA, Canada and Australia), Country X  }
% The Corresponding Author should be marked with an asterisk
% Provide the exact contact address (this time including street name and city zip code) and email of the corresponding author
\def\corrAuthor{Corresponding Author}

\def\corrEmail{email@uni.edu}

\begin{document}
\onecolumn
\firstpage{1}

\title{5\sprime\ untranslated regions of viral mRNAs are repurposing targets} 

\author[\firstAuthorLast ]{\Authors} %This field will be automatically populated
\address{} %This field will be automatically populated
\correspondance{} %This field will be automatically populated

\extraAuth{}% If there are more than 1 corresponding author, comment this line and uncomment the next one.
%\extraAuth{corresponding Author2 \\ Laboratory X2, Institute X2, Department X2, Organization X2, Street X2, City X2 , State XX2 (only USA, Canada and Australia), Zip Code2, X2 Country X2, email2@uni2.edu}


\maketitle


\begin{abstract}

%%% Leave the Abstract empty if your article does not require one, please see the Summary Table for full details.
\section{}
For full guidelines regarding your manuscript please refer to \href{http://www.frontiersin.org/about/AuthorGuidelines}{Author Guidelines}. 

As a primary goal, the abstract should render the general significance and conceptual advance of the work clearly accessible to a broad readership. References should not be cited in the abstract. Leave the Abstract empty if your article does not require one, please see \href{http://www.frontiersin.org/about/AuthorGuidelines#SummaryTable}{Summary Table} for details according to article type. 


\tiny
 \keyFont{ \section{Keywords:} keyword, keyword, keyword, keyword, keyword, keyword, keyword, keyword} %All article types: you may provide up to 8 keywords; at least 5 are mandatory.
\end{abstract}

\section{Introduction}
Small-molecule (SM) targeting of RNA–protein complexes demands an infusion of innovation to accelerate the discovery of new modalities that both illuminate biological mechanisms and hold therapeutic promise.
RNA–protein (RNP) complexes orchestrate essential steps in gene expression under both normal physiological and pathological conditions.  
Yet despite decades of research and substantial  investment, the repertoire of SM modulators of RNPs remains limited.  
To bridge this gap, the field requires complementary strategies that operate in parallel with sequencing-based approaches that map RNA structure, RNA-binding protein (RBP) interactions, and SM binding sites across the transcriptome.

Bridging the discovery gap will require integrative frameworks that combine structural prediction, molecular dynamics (MD) simulations, and experimental workflows capable of quantifying atomic fluctuations and interaction mechanisms that define RNP binding modes. 
Recent advances---exemplified by AlphaFold3---now enable modeling of RNP complexes within intact RNA tertiary architectures, yielding binding interfaces that more closely recapitulate native conformations than those captured in experimental structures of short RNA fragments.  
When complemented by  MD simulations and biophysical interrogation, these predicted complexes provide a robust foundation for generating dynamic RNP ensembles suitable for virtual  screening.  
Together, this hybrid framework links predictive modeling with empirical observation, yielding an experimentally anchored view of RNP structure that captures the transient and allosteric behaviors most relevant to function.

Selecting the most suitable RNP interface to pursue as a drug target is challenging, given that a majority of expressed human transcripts---estimated at over 50\%---exhibit measurable RBP occupancy in vivo, as demonstrated by transcriptome-wide CLIP and RNA interactome capture studies \citep{yeo2020}. 
For viruses, the prevalence of RBP-RNA interactions are predicted to be even higher, as both viral and cellular proteins engage viral RNA genomes at nearly every stage of the replication cycle \citep{kim2021,routh2022}.
During certain phases of the viral life cycle the abundance of both protein and RNA components may also be elevated far above those of cellular components. This pervasive binding creates a crowded and dynamic molecular environment that complicates the identification of discrete, druggable RNP interfaces.

\begin{figure}
    \begin{center}
    \includegraphics[width=\linewidth]{figures/5p-utr-targets.pdf}
    \caption{The \fputr\ constitutes an attack surface replete with targets for SMs perturbing the translation process. Because RBPs are needed to modulate translation for the purposes of the viral life-cycle, any ligand perturbing their affinity for the RNA-element they interact with has potential for antiviral activity. An SM may do this by binding the RBP in an allosteric site (Target class 1), competing directly for the RNA-element with the RBP (Target class 2), perturbing modifying enzymes that act on the \fputr\ (Target class 3) or RBP (target class 4) or finally SMs that bind across the RNA-protein interface, preorganizing it to prefer binding to one factor over another (Target class 5).}\label{fig:5p-utr-targets}
    \end{center}
\end{figure}

Among cellular and viral RNA elements, 5\sprime untranslated regions (\fputr s) constitute rich and underexplored regulatory hubs with emerging potential as targets for SM modulation (Fig. \ref{fig:5p-utr-targets}). 
\fputr s integrate structural motifs, chemical modifications, and protein interactions that collectively fine-tune ribosome recruitment, translation efficiency, and cellular responses to stress and signaling cues. 
Their architectures are remarkably diverse, often folding into conserved secondary structures interspersed with bulges, junctions, and internal loops that serve as docking platforms for ribosomes, other RNA elements, regulatory proteins, and viral factors. 
Furthermore, dysregulation of \fputr-mediated control is increasingly implicated in cancer, neurodegeneration, and viral pathogenesis, underscoring their functional importance \citep{hentze2012}.


From a therapeutic standpoint, the modularity and structural plasticity of \fputr s make them compelling drug targets. 
Small molecules that modulate specific RBP-RNA interfaces could reprogram translation with predictable outcomes. 
Visualizing the conformational and RBP-interaction landscapes of \fputr s at atomic resolution may reveal cryptic regulatory pockets and broaden the spectrum of RNP assemblies accessible to rational SM design.

Within these regulatory regions, bulge loops emerge as recurrent and functionally significant RNA structural motifs. 
Local deviations from canonical helices expose unpaired nucleotides that frequently serve as recognition elements for RBPs and other ligands. 
The conformational plasticity of bulge loops enables diverse binding modes, making them common interaction sites for RBPs involved in splicing, translation, and viral replication. 
Consistent with this, SM-RNA interaction databases such as R-BIND identify bulge-containing motifs among the most prevalent structural classes engaged by bioactive ligands \citep{hargrove2022}. 
Structural surveys further indicate that bulge loops constitute roughly 20-30\% of local secondary-structure motifs across cellular and viral RNAs, reflecting their prevalence in dynamic, ligand-accessible architectures \citep{patel2000}. 
Collectively, these observations highlight bulge loops as privileged structural motifs for molecular recognition and as promising entry points for drug discovery targeting structured RNAs and RNP complexes.

Integrating predictive and experimental structural data provides a powerful foundation for rational drug repurposing. 
While working with approved chemical matter accelerates any regulatory process downstream of an identified binder, identifying binders against a desired target becomes challenging. 
If the `library' is to be the set of approved molecules, the question becomes ``is the hit even in the library.'' 
However, because of the multivalent and dynamic nature of \fputr s, those associated with disease are a target rich environment for modulators. 
And because the disease condition materially elevates the abundance of these targets above that of cellular components, lower affinity molecules are viable, and may even be desirable, as therapies. 
Finally, when it comes to transcription and translation performance, small deviations can alter the stoichiometry of disease-causing products enough to perturb disease physiology \citep{butcher2012}.

Structural ensembles generated through MD simulations initiated from AlphaFold3 models, and validated by NMR spectroscopy, can reveal transient binding pockets and allosteric surfaces that remain hidden in static structures. 
These dynamic ensembles can be exploited to screen and prioritize approved small molecules whose physicochemical properties favor engagement with RNP interfaces. 
Leveraging the vast chemical diversity of existing FDA approved pharmacophores accelerates the identification of modulators that stabilize or disrupt specific RNP conformations---enabling mechanism-guided repurposing that shortens development timelines and de-risks early discovery. 
Ultimately, the convergence of AI-driven structure prediction, MD simulation, and biophysical experimentation establishes a tractable and scalable path toward targeting RNPs with small molecules.

Here, we advance a conceptual framework integrating structure prediction, molecular simulation, and experimental validation to evaluate the druggability of RNP complexes. We discuss this framework as applied to the \fputr of Enterovirus A-71 (EV-A71) as a representative example. 
This region contains an internal ribosome entry site (IRES) that engages more than 20 host RBPs and a viral-derived small RNA to modulate translation. 
Perturbations that weaken these RBP-IRES interactions significantly reduce viral titers by compromising cap-independent translation. 
Multiple RNA-focused screens have identified small molecules that suppress EV-A71 replication, with DMA-135 emerging as a selective modulator of a key IRES-protein interface. 
These observations underscore the potential for integrative structural and computational methodologies to pinpoint mechanistic vulnerabilities in viral RNA regulation and accelerate the rational repurposing of clinically relevant compounds for antiviral intervention.

\section{Structured and conserved \fputr are an underexploited attack surface for many diseases}

Conceptually, there are at least five types of interactions present in functional \fputr s that could be modulated by noncovalent binders---of these, targeting binders to the RNP interface or interaction sites on the RNA itself are most appealing (Sudeshi).

RNA and RNA-protein interaction sites also exist in viral RNAs, such as the \fputr of EV71.

\section{There is evidence that FDA approved molecules bind to RNAs and modulate RNA-protein complexes}

The activity of RNA is tightly regulated by RNA binding proteins (RBPs) through sequence and/or structural recognition \citep{Gerstberger2014,Dominguez2018}. Dysregulation or off-target binding of RBPs is associated with neurodegenerative diseases \citep{Bryce-Smith2025,Zeng2025}, cancer \citep{Bell2013}, and developmental disorders \citep{Darnell2011}. Amyotrophic lateral sclerosis (ALS) is a neurodegenerative disease characterized by an abnormal accumulation of fused in sarcoma (FUS) protein in the cytoplasm. To minimize the accumulation of cytoplasmic FUS, existing cellular FUS proteins enagage pre-mRNA, producing a feedback loop in the presence of excess FUS protein, thus halting the expression of new FUS proteins \citep{Ayala2011}. Consequently, the disruption of the FUS-pre-messenger RNA (mRNA) interaction exacerbates ALS progression. Another neurodegenerative disease, frontotemporal dementia (FTD) is characterized by the depletion of the mRNA splicing protein, TDP-43, in the nucleus of neurons. Recent studies have described an alternate functional role for TDP-43; TDP-43 interacts with GU rich repeat sequences in mRNAs to modulate cleavage and alternative polyadenylation (APA) \citep{Bryce-Smith2025,Zeng2025}. Improper regulation of APA initiates a cascade of events that results in the loss of functional proteins for axonal regeneration/repair \citep{Melamed2019}. These examples highlight the integral role of proper ribonucleoprotein (RNP) regulation in the maintenance of optimal cellular function.

Enhanced specificity of RBP interactions can be achieved by cooperative binding of RNA recognition motifs (RRM) within RBPs to their cognate RNA binding sites \citep{Mackereth2011,Hennig2015,Duszczyk2022}. 
In addition, endogenous small molecules such as adenosine triphosphate (ATP), guanosine triphosphate (GTP), and S-adenosylmethionine (SAM) regulate various cellular processes such as RNA localization, translation, and mRNA processing via mediation of RBP-RNA interactions \citep{Miao2025}. 
Non-endogenous small molecules have also emerged as an attractive tool for modulating RBP-RNA interactions. 
In 2020, the splice modulator risdiplam was approved by the Food and Drug Administration (FDA), serving as the only non-ribosomal RNA-targeting FDA approved small molecule to date \citep{Mullard2020,Ratni2021}. 
Risdiplam stabilizes the interaction between U1 small nuclear ribonucleoprotein and survival motor neuron (SMN) mRNA, leading to replenished SMN2 protein levels for proper maintenance of motor neurons \citep{Ratni2018,Ratni2021}. 
Similarly, the amiloride DMA-135 stabilizes the RNP interaction between AUF1 and stem loop 2 of EV-A71 internal ribosomal entry site (IRES), thereby repressing cap-independent translation of important viral proteins \citep{Davila-Calderon2020,Davila-Calderon2024}. However, the expediency in this approach is tainted by the number of RNA-targeting small molecules that have been approved by the FDA.
 
The dearth in RNA-targeting FDA-approved small molecules can partly be ascribed to the conformational heterogeneity of RNAs, as well as the relatively fewer RNA structures deposited in the Protein Data Bank (approximately 26 protein structures for every single RNA structure).
Encouragingly, recent RNA-small molecule screening campaigns have identified some protein inhibitors that bind RNAs as well. 
A few examples include: mitoxantrone, a topoisomerase II inhibitor that binds to the oncogenic pre-miR-21 RNA \citep{Velagapudi2018}. 
Another example is the selective estrogen receptor modulator (SERM) raloxifene that also binds to the \fputr\ of the hHBV \textbf{\(\epsilon\)} RNA \citep{LeBlanc2022}. 
In fact, it was previously shown that several protein-binding small molecules possess significant chemical structure similarities to RNA-binders, hinting at mechanisms by which these drugs confer side-effects \citep {Fang2023}. 

Levofloxacin is an FDA-approved oral antibiotic that functions as an inhibitor of the bacterial enzymes, DNA gyrase and topoisomerase IV \citep{Croom2003}. 
However, a few common side effects associated with the consumption of this drug include: nausea, neuropathy, and difficulty breathing. 
In one case study, Fang et al demonstrated that Levofloxacin has pervasive interactions throughout the transcriptome. 
In addition, they demonstrated that RNA-Levofloxacin interactions were not only biochemically driven, but had functional relevance in the context of cellular output \citep{Fang2023}. 
Levofloxacin interacts with a G4 motif to compromise translation of the Y-box binding protein 1 (YBX1) mRNA, possibly contributing to the side effects observed with Levofloxacin consumption. 
Off-target binding of Levofloxacin to the \fputr\ of YBX1 mRNA alters the structural dynamics of certain regions in the RNA, potentially modulating the loading of translation initiation factors \citep{Fang2023}. 
Similarly, FDA-approved aminoglycosides such as paromomycin function by binding to ribosomal RNAs to repress or compromise the expression of essential bacterial proteins \citep{Fourmy1996,Recht1996,Vicens2001}. 
Therefore, it may be worth exploring the possibility of repurposing Levofloxacin to target pathogens with G4 motifs that are pivotal to their life cycles. 
By extension, we propose screening and repurposing FDA-approved drugs for RNA/RNP targets. 
This approach should significantly reduce the cost, labor, and time involved in the drug development process.


\section{There is a path to virtually screening many potential targets within a \fputr\ of interest}

\begin{figure}
    \begin{center}
    \includegraphics[width=0.8\linewidth]{figures/workflow-virtual-expt-5putr-screen.pdf}
    \caption{An illustration of the workflow we propose. Using the literature, RNA-elements and their protein binding partners in the \fputr s of disease relevant mRNAs are identified. The RNA elements are grafted into a designed fragment using computational tools. These models are validated using rapid experimental means such as NMR, and simulated with molecular dynamics to model the ensemble of the target. Using pocket finding tools and ensemble virtual screening with a library of approved molecules, we prioritize SMs to validate experimentally. SMs that show biophysical and infectivity assay activity become antiviral treatment options because of their approved nature.}\label{fig:workflow}
    \end{center}
\end{figure}

Considering the \fputr of EV71 as a representative example, we can identify XXX targetable RNA elements, and YYY RNP complexes (Barrington).

We can build models of these complexes using AlphaFold3.
For example we built three-dimensional structural models of protein-RNA complexes were made using AlphaFold 3 (AF3) \citep{jumper2024}.
As a test, we tried input sequences that included the RRM1 domain of AUF1 and the RRM12 domain of UP1, and a minimal RNA nucleotide sequences that mimics the fold of the SL2 RNA bulge.  
The generated models were evaluated based on AF3's self-reported confidence metrics, primarily the interface predicted TM-score (ipTM) and the overall predicted TM-score (pTM). 
For the UP1-minimal RNA fragment, we obtained a selected model with an ipTM of 0.66 and pTM of 0.81; and AUF RRM1 with the SL2 GAGA RNA fragment, resulting in a selected model with an ipTM of 0.66 and pTM of 0.71, suggesting our models have tolerable confidence levels overall.
A local confidence analysis using the predicted local distance difference test (pLDDT) score for both models indicated 'Very high' confidence (pLDDT > 90) for core protein domains, while RNA molecules and flexible protein loops exhibited more variable confidence, ranging from 'Confident' (70 < pLDDT < 90) to 'Low' (50 < pLDDT < 70).

Using the structural fragment strategy outlined in figure 2, we constrain the AF3 prediction to the most probable protein-RNA interface.  
This targeted approach increases the likelihood that the predicted complex captures the native binding mode, as most RNA binding proteins recognize sequence-specific motifs presented within single-stranded regions of structured RNAs \citep{patel2000,hentze2012,yeo2020}. 
By isolating the relevant RNA element  as a modular bulge fragment, we focus the interaction surface, while the RBP engages the target through its canonical B-sheet interface.   
In parallel, we can prepare isotopically enriched protein and RNA to validate the  binding mode (Fig 2). 
The modular design of the RNA structural fragment enables selective isotopic labeling of nucleotides within the bulge against a background of unlabeled nucleobases engaged in Watson-Crick base-pairs (in this case, GC pairs). 
This feature allows clear detection of ligand induced chemical shift perturbations without requiring full chemical shift assignments. 
From a screening perspective, this approach allows facile and efficient detection of ligand-induced changes of the NMR signals arising from the bulge loop environment.

To perform a virtual screen, we'd first run simulations of these complexes looking for pockets or other interfaces.
Although simulations are a medium-low throughput technique, they can still help validate AF3 models by permitting physics to dictate whether conformations in the initial model are maintained.
Using either adaptive sampling, or some other exploratory enhanced sampling such as Gaussian-accelerated Molecular Dynamics, gives a rough sketch of the ensemble of the complex on a 'several weeks' timeline for common compute resources \citep{defabritiis2016,miao2021}.
Shorter timescale simulations also permit the detection of cryptic pockets, which can often appear within hundreds of nanoseconds of replicate simulation (microseconds, aggregate)\citep{bowman2023}.
For identifying pockets, ligsite or f-pocket can be used, as well as the RNA specific update proposed by \citet{weeks2025} \citep{barnickel1997,tuffery2009}.

Docking would then be performed versus the ensembles that exhibit pockets using the Broad drug repurposing hub database, and using PopShift to aggregate the docking results \citep{golub2017,bowman2024}.
Virtual screening versus an ensemble is important because complexes involving RNA result in dynamic pockets \citep{al-hashimi2019}.
Even typically `rigid' proteins exhibit pocket dynamics \citep{gilson2017, smith2018}.
PopShift addresses some issues with ensemble docking by using a discretized model of the ensemble (usually a Markov State Model (MSM)) to organize the receptor conformations to be docked to \cite{bowman2024}.
Because PopShift scores which receptor conformations are preferred by different ligands, it can be used to understand rudimentary allosteric effects sampled by the ligand-free ensemble.

To reliably validate RNA-small molecule, protein-small molecule interactions or modulators of RNA-protein complexes, a rigorous characterization pipeline should be employed. 
A well-designed biophysical framework ideally would explore interactions using orthogonal methods, leading to a validation that is comprised of structural, thermodynamic, and kinetic information at its core, providing a strong rationale in advancing hits as candidates for additional exploration as alternative therapies. 
Exploring interactions is routinely performed using well-established biophysical techniques such as X-ray crystallography, nuclear magnetic resonance (NMR) spectroscopy, and isothermal titration calorimetry (ITC). 
When applicable, these methods provide highly valuable structural and thermodynamic insights.
However, for complex or dynamic systems, usually a single technique is insufficient; instead, a combination of orthogonal approaches is required to obtain converging, reliable interpretations.

X-ray crystallography is often considered the gold standard for high-resolution structural characterization. However, obtaining well-diffracting crystals is frequently challenging, particularly for flexible and heterogeneous complexes. Additionally, crystallographic structures typically represent 1 of several local minima with functional relevance, which doesn’t completely reflect the conformational ensemble that governs biologically relevant interactions.

In such cases, solution-based spectroscopy, particularly NMR, can provide valuable complementary information. NMR is especially useful when characterizing RNA, proteins, and small-molecule interactions in solution, offering insights into binding specificity, site-specific interactions via chemical shift perturbations, and kinetics. These experiments can rapidly identify and characterize interfaces governing complex formation. However, NMR is often accompanied with molecular size limitations, increased sample concentration requirements, and spectral overlap.

ITC is equally valuable for validating complex formation and quantifying binding affinity, stoichiometry, and thermodynamic signatures (enthalpic vs. entropic contributions). However, purely entropy-driven interactions or weak binding events may be difficult to characterize, especially when heat changes are small. This limitation underscores the need for orthogonal validation. 

Additional complementary approaches include fluorescence anisotropy, differential scanning fluorimetry (DSF), microscale thermophoresis (MST), and surface plasmon resonance. Each of these methods can help evaluate binding specificity, kinetics, and ligand-induced stabilization of RNA motifs or RNA-protein complexes. In many cases, combining structure-based methods (NMR, crystallography), thermodynamic measurements (ITC), and high throughput screening-compatible assays (MST, SPR, DSF) offers an in dept and comprehensive assessment.
In summary, designing an effective experimental strategy for validating interactions between small-molecules and RNA/protein requires a combination of complementary biophysical techniques. Each method provides unique strengths but also carries inherent limitations; therefore, an orthogonal, multi-method approach is essential to obtaining reliable structural and mechanistic insights.


\section{Discussion and Conclusion}

\section*{Conflict of Interest Statement}
%All financial, commercial or other relationships that might be perceived by the academic community as representing a potential conflict of interest must be disclosed. If no such relationship exists, authors will be asked to confirm the following statement: 

The authors declare that the research was conducted in the absence of any commercial or financial relationships that could be construed as a potential conflict of interest.

\section*{Author Contributions}

The Author Contributions section is mandatory for all articles, including articles by sole authors. If an appropriate statement is not provided on submission, a standard one will be inserted during the production process. The Author Contributions statement must describe the contributions of individual authors referred to by their initials and, in doing so, all authors agree to be accountable for the content of the work. Please see  \href{https://www.frontiersin.org/about/policies-and-publication-ethics#AuthorshipAuthorResponsibilities}{here} for full authorship criteria.

\section*{Funding}
Details of all funding sources should be provided, including grant numbers if applicable. Please ensure to add all necessary funding information, as after publication this is no longer possible.

\section*{Acknowledgments}
This is a short text to acknowledge the contributions of specific colleagues, institutions, or agencies that aided the efforts of the authors.

\section*{Supplemental Data}
 \href{http://home.frontiersin.org/about/author-guidelines#SupplementaryMaterial}{Supplementary Material} should be uploaded separately on submission, if there are Supplementary Figures, please include the caption in the same file as the figure. LaTeX Supplementary Material templates can be found in the Frontiers LaTeX folder.

\section*{Data Availability Statement}
The datasets [GENERATED/ANALYZED] for this study can be found in the [NAME OF REPOSITORY] [LINK].
% Please see the availability of data guidelines for more information, at https://www.frontiersin.org/about/author-guidelines#AvailabilityofData

\bibliographystyle{Frontiers-Harvard} %  Many Frontiers journals use the Harvard referencing system (Author-date), to find the style and resources for the journal you are submitting to: https://zendesk.frontiersin.org/hc/en-us/articles/360017860337-Frontiers-Reference-Styles-by-Journal. For Humanities and Social Sciences articles please include page numbers in the in-text citations 
%\bibliographystyle{Frontiers-Vancouver} % Many Frontiers journals use the numbered referencing system, to find the style and resources for the journal you are submitting to: https://zendesk.frontiersin.org/hc/en-us/articles/360017860337-Frontiers-Reference-Styles-by-Journal
\bibliography{repurposing-vs-5p-utr,repurposing-vs-5p-utr-lgs}

%%% Make sure to upload the bib file along with the tex file and PDF
%%% Please see the test.bib file for some examples of references

% \section*{Figure captions}

%%% Please be aware that for original research articles we only permit a combined number of 15 figures and tables, one figure with multiple subfigures will count as only one figure.
%%% Use this if adding the figures directly in the mansucript, if so, please remember to also upload the files when submitting your article
%%% There is no need for adding the file termination, as long as you indicate where the file is saved. In the examples below the files (logo1.eps and logos.eps) are in the Frontiers LaTeX folder
%%% If using *.tif files convert them to .jpg or .png
%%%  NB logo1.eps is required in the path in order to correctly compile front page header %%%

% \begin{figure}[h!]
% \begin{center}
% \includegraphics[width=10cm]{logo1}% This is a *.eps file
% \end{center}
% \caption{ Enter the caption for your figure here.  Repeat as  necessary for each of your figures}\label{fig:1}
% \end{figure}

% \setcounter{figure}{2}
% \setcounter{subfigure}{0}
% \begin{subfigure}
% \setcounter{figure}{2}
% \setcounter{subfigure}{0}
%     \centering
%     \begin{minipage}[b]{0.5\textwidth}
%         \includegraphics[width=\linewidth]{logo1.eps}
%         \caption{This is Subfigure 1.}
%         \label{fig:Subfigure 1}
%     \end{minipage}  
   
% \setcounter{figure}{2}
% \setcounter{subfigure}{1}
%     \begin{minipage}[b]{0.5\textwidth}
%         \includegraphics[width=\linewidth]{logo2.eps}
%         \caption{This is Subfigure 2.}
%         \label{fig:Subfigure 2}
%     \end{minipage}

% \setcounter{figure}{2}
% \setcounter{subfigure}{-1}
%     \caption{Enter the caption for your subfigure here. \textbf{(A)} This is the caption for Subfigure 1. \textbf{(B)} This is the caption for Subfigure 2.}
%     \label{fig: subfigures}
% \end{subfigure}

%%% If you don't add the figures in the LaTeX files, please upload them when submitting the article.
%%% Frontiers will add the figures at the end of the provisional pdf automatically
%%% The use of LaTeX coding to draw Diagrams/Figures/Structures should be avoided. They should be external callouts including graphics.

\end{document}
